### DISCUSSION TITLE
How to change the color of the play button on hover?

### DISCUSSION THREAD

**Discussion by xaviernouaille** [muxinc/next-video#249] [Q&A]
Hi,

How to change the color of the play button on hover? There are plenty of shadow-roots. It's very tedious and I can't change the value. How to do it? 

Thanks a lot for your help.

<img width="1407" alt="Capture d’écran 2024-05-02 à 00 20 22" src="https://github.com/muxinc/next-video/assets/68466322/b3a34fa3-5692-48bb-b73b-6f1eb0529864">

**Comment by luwes** [comment]
next-video uses Mux player so all configurations documented below should work
https://docs.mux.com/guides/player-customize-look-and-feel#provide-color-variables

**Reply by xaviernouaille** [reply]
Thanks you rock
